styles.css

```diff
@@ -22,32 +26,33 @@
 
   h1 {
     font-size: 3.125rem;
-    font-family: 'Playfair Display', serif;
+    font-family: "Playfair Display", serif;
     line-height: 100%;
     font-weight: 400;
   }
 
   h2 {
     font-size: 3.125rem;
-    font-family: 'Playfair Display', serif;
+    font-family: "Playfair Display", serif;
     line-height: 100%;
     font-weight: 400;
   }
 
   #headline h1 {
     margin: 0 1rem;
-    word-spacing: 30rem;
+    word-spacing: 100rem;
```

Comment:
- [ ] The headline margins need to be close to the Figma design, kindly work on that issue.

![rev1](https://user-images.githubusercontent.com/53347966/230517842-555aac14-dfde-4747-a57e-a67882971fae.JPG)